src/__tests__/Autocomplete.test.tsx

```diff
@@ -457,11 +320,145 @@ describe('Autocomplete', () => {
 
   describe('null context', () => {
     it('throws errors when Autocomplete context is null', () => {
+      const spy = jest.spyOn(console, 'error').mockImplementation(() => {})
+
       expect(() => HTMLRender(<Autocomplete.Overlay />)).toThrowError('AutocompleteContext returned null values')
       expect(() => HTMLRender(<Autocomplete.Input />)).toThrowError('AutocompleteContext returned null values')
       expect(() => HTMLRender(<Autocomplete.Menu items={mockItems} selectedItemIds={[]} />)).toThrowError(
         'AutocompleteContext returned null values'
       )
+
+      spy.mockRestore()
+    })
+  })
+})
+
+describe('snapshots', () => {
```

Comment:
@mperrotti yeah, for a super odd reason too 🤔 Let me know if you have any ideas!

> In order to get the snapshot tests and testing-library tests working together, I needed to split them up into a separate test suite. Otherwise, the mismatch act() warning would not go away. Let me know if there is a better approach when everything is in one test suite, I couldn't figure out a way to make it work 🤔